js/src/components/contentAnalysis/KeywordInput.js

```diff
@@ -90,10 +93,12 @@ KeywordInput.propTypes = {
 	onFocusKeywordChange: PropTypes.func.isRequired,
 	onFocusKeyword: PropTypes.func.isRequired,
 	onBlurKeyword: PropTypes.func.isRequired,
+	isSEMrushIntegrationActive: PropTypes.bool,
 };
 
 KeywordInput.defaultProps = {
 	keyword: "",
+	isSEMrushIntegrationActive: false,
```

Comment:
Shouldn't the default be true? Unless I'm missing something.

Reply:
See above: if we don't get a value for the prop isn't it better to prevent the modal to be rendered?